newdle/client/src/setupProxy.js

```diff
@@ -6,6 +6,7 @@ const proxy = require('http-proxy-middleware');
 module.exports = app => {
   const config = {target: process.env.FLASK_URL || 'http://127.0.0.1:5000'};
   app.use(proxy('/api/', config));
+  app.use(proxy('/config/', config));
```

Comment:
Just put your new endpoint in `/api/config` - no need to change the proxy settings that way